content/waku/2024-gantt.md

```diff
@@ -22,94 +22,139 @@ gantt
     axisFormat %d-%b
     weekday monday
     
+%% Team legend:
+%% task name: team (accountable person)
+%% - r-*: research (jm-clius)
+%% - t-*: telemetry (fryorcraken)
+%% - n-*: nwaku (ivansete)
+%% - j-*: js-waku (weboko)
+%% - c-*: chat (plopezlpz)
+%% - g-*: go-waku (fryorcraken)
+%% - bd-*: BD (pedro)
+%% - se-*: Solution Engineer (vpavlin)
```

Comment:
For the next iteration I want the team leads to handle the Gantt Chart so set some names.

Which means everyone mentioned in this legend can check:

(1) In the _Milestones overview with deliverables_ section, that every deliverable needed work from their team has a Gantt task with the right prefix after the keyword `after`. For example:

`E2e reliability protocol Status integration: milestone, crit, after r-e2e-rel-status c-e2e-rel-status, 0`
has work scheduled for research and chat team, but no work for js-waku and nwaku teams

(2) Then, ensure that the assignment of tasks to team members is roughly correct until 31 Dec 2024. What matters is the completion dates, as it is what is then committed to in roadmap.